src/main/java/pl/simpleascoding/tutoringplatform/domain/token/TokenType.java

```diff
@@ -1,5 +1,5 @@
 package pl.simpleascoding.tutoringplatform.domain.token;
 
 public enum TokenType {
-    REGISTER, PASSWORD, EMAIL
+    REGISTER, PASSWORD, EMAIL, CHANGE_EMAIL_CONFIRMATION
```

Comment:
This enum name implies, that it is used for changing the user's email address. Please rename to something like `CHANGE_PASSWORD_CONFIRMATION`.
Also, if we're not using `PASSWORD` and `EMAIL` values, you can delete them.
I'd also advise changing the `REGISTER` enum to `REGISTER_CONFIRMATION`, so that all enums in this file have similar names.